### PR TITLE
Budget96 info cleanup and updates

### PR DESCRIPTION
**Pull request by DonutCables**
## Description

Updates to basic info files for Budget96 for consistency/cleanliness. Includes adding proper ID/description values for VIA or other interaction in the future. 

Also includes a personal map folder.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — COMMENTED** [review]
Could you convert the tabs to spaces in the keymap? 
You can run `qmk cformat keyboards/donutcables/budget96/keymaps/donut/keymap.c` to do this.

**Comment by DonutCables** [comment]
> 
> 
> Could you convert the tabs to spaces in the keymap?
> You can run `qmk cformat keyboards/donutcables/budget96/keymaps/donut/keymap.c` to do this.

Accidentally bodged my rebase attempt and didn't know a reset would close the PR. Should I open a new one or can this one be pointed to the new commits?

**Review by drashna — APPROVED** [review]

File: keyboards/donutcables/budget96/config.h [hunk 1]
```diff
@@ -19,11 +19,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-#define VENDOR_ID       0x20A0
-#define PRODUCT_ID      0x422D
-#define DEVICE_VER      0x0200
+#define VENDOR_ID       0xDCDC
+#define PRODUCT_ID      0x0002
+#define DEVICE_VER      0x0001
 #define MANUFACTURER    DonutCables
-#define PRODUCT         budget96
+#define PRODUCT         Budget96
+#define DESCRIPTION		A budget 96key
```
Comment (fauxpark) [inline keyboards/donutcables/budget96/config.h:27]:
```suggestion
```

File: keyboards/donutcables/budget96/keymaps/donut/keymap.c [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+	KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case xkcd:
+            if (record->event.pressed) {
+                SEND_STRING("https://xkcd.com/927/");
+            } else {
```
Comment (fauxpark) [inline keyboards/donutcables/budget96/keymaps/donut/keymap.c:90]: The empty `else` blocks here and the `break`s can be removed, as they are dead code.

File: keyboards/donutcables/budget96/keymaps/donut/rules.mk [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+SPACE_CADET_ENABLE = no
+GRAVE_ESC_ENABLE = no
+ENCODER_ENABLE = no
+TAP_DANCE_ENABLE = no
+MIDI_ENABLE = no 
+HD44780_ENABLE = no
+BLUETOOTH_ENABLE = no
+FAUXCLICKY_ENABLE = no
+NKRO_ENABLE = no
+BOOTMAGIC_ENABLE = no
```
Comment (fauxpark) [inline keyboards/donutcables/budget96/keymaps/donut/rules.mk:15]:
```suggestion
VELOCIKEY_ENABLE = yes
UNICODE_ENABLE = yes
BACKLIGHT_ENABLE = no
MOUSEKEY_ENABLE = no
COMMAND_ENABLE = no
SPACE_CADET_ENABLE = no
GRAVE_ESC_ENABLE = no
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

File: keyboards/donutcables/budget96/config.h [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Copyright 2017-2020 Luiz Ribeiro <luizribeiro@gmail.com>, DonutCables <contact@donutcables.com>
```
Comment (fauxpark) [inline keyboards/donutcables/budget96/config.h:2]:
```suggestion
Copyright 2020 DonutCables <contact@donutcables.com>
```

**Review by fauxpark — APPROVED** [review]

**Pull request merged**